scripts/lib.sh

```diff
@@ -65,53 +75,34 @@ gpgcheck=1
 enabled=0
 gpgkey=file://${CONFIG_DIR}/RPM-GPG-KEY-CentOS-7
```

Comment:
This should use a variable too. Maybe add a `GPGKEY_FILE` variable to `config.sh` or similar.